lib/angular/shared/geolocation/geolocation-spec.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+/* globals inject */
+/*global navigator:true*/
+
+var sinon = require('sinon'),
+  angular = require('angular'),
+  //should = require('should'),
+  initModule = require('../../../../test/initModule');
+require('angular-mocks');
+
+describe("Test Geolocation Service", function() {
+  //Setting up a mock module to test.
+  before(function() {
+    initModule();
+    // Import the geolocation service
+    require('./geolocationService');
+  });
+
+  beforeEach(angular.mock.module('wfm.appform'));
+  beforeEach(inject(function(GeolocationService) {
+    this.GeolocationService = GeolocationService;
+  }));
+
+  it('The getGeolocation service method should return coordinates when navigator is available and an error message when not available', function() {
+
+    // setup stub
+    var sinonStub = sinon.stub();
```

Comment:
Can you add a more specific stub name. `sinonStub` is too generic. Something like `getCurrentPositionStub`